components/introduction.js

```diff
@@ -2,26 +2,26 @@ import styled from "styled-components";
 
 export default function Introduction() {
   return (
-    <>
-      <p>
-        <mark>
-          Auf folgenden Seiten versammelt sich eine Auswahl ehemaliger
-          Kultusstätten Hamburgs, die mittlerweile profaniert sind. Bei dieser
-          Zusammenstellung kommt es mir dabei nicht auf eine bestimmte
-          Definition von Glaube an, so finden sich hier frühere jüdische,
-          christliche und heidnische Plätze.
-          <br></br>
-          Alle Stellen gebe ich mit ihren geografischen Koordinaten an, dazu
-          alte und neue Ortsbezeichnungen. Mit meinen Fotografien möchte ich
-          gerne die Stimmung der heutigen Plätze festhalten – lässt sich hier
-          noch etwas Kultisches erahnen? Sie stellen keine Übersicht über den
-          jetzigen Zustand dar. Ihnen entgegen habe ich historisches
-          Bildmaterial gesetzt, das den zeitlichen Wandel der jeweiligen Orte
-          zeigt. Die Dokumente habe ich möglichst so herausgesucht, dass sie uns
-          etwas über den Grund der Profanierung verraten. Im Kontrast dazu
-          stehen die ausgewählten Zitate.
-        </mark>
-      </p>
-    </>
+    <p className="introduction">
+      <StyledSpan>
+        Auf folgenden Seiten versammelt sich eine Auswahl ehemaliger
+        Kultusstätten Hamburgs, die mittlerweile profaniert sind. Bei dieser
+        Zusammenstellung kommt es mir dabei nicht auf eine bestimmte Definition
+        von Glaube an, so finden sich hier frühere jüdische, christliche und
+        heidnische Plätze.
+        <br></br>
```

Comment:
try not to use br, there should be seperately styled text elements in best case. 
also you can just wirte </br> ;)